packages/paste-core/core-bundle/package.json

```diff
@@ -12,6 +12,53 @@
   "publishConfig": {
     "access": "public"
   },
+  "nx": {
```

Comment:
Core bundle needs it's own nx config overrides as some of the scripts are different and how we build it is different.